### ISSUE TITLE
Will not install in Addons in 2.93 Preferences

### ISSUE THREAD

**Issue by ga1424** [PhysicalAddons/physical-starlight-and-atmosphere#54]
Removed Ver, 1.3 and Downloaded Ver. 1.4.2.  

When trying to activate after install message reads   "Traceback (most recent call list):"

Should I have deleted 1.3 before installing 1.4?

**Comment by karlisup** [comment]
Hi @ga1424,
TLDR;
Unfortunately yes. That is [the only way](https://www.physicaladdons.com/psa/updating/#troubleshooting) we can ensure that the addon will start correctly after updating to the new version.

I have multiple times tried to figure out a way for the script to recognize that a new version is applied and it should find a way to update without returning an error, but so far unsuccessful. Recently I tried using CGCookie blender-addon-updater which removes addon, adds backup before installing a new version of addon on top of it, but the experience was not stable. Sometimes it was without errors, but sometimes they were there. So this is an ongoing process and until we find a good solution for this issue, we do encourage removing the previous version before applying the new one.
